src/librustc_errors/diagnostic.rs

```diff
@@ -11,7 +11,6 @@
 use CodeSuggestion;
 use Level;
 use RenderSpan;
-use RenderSpan::Suggestion;
```

Comment:
I left it in for keeping the diff small. I convert  to it in the emitter and json impls. Wanted to get feedback on a readable diff first